core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -876,7 +882,8 @@ public synchronized void stopInternal() throws InterruptedException, IOException
    */
   public synchronized CompletableFuture<RaftClientReply> sendMessageAsync(
       RaftPeerId server, Message message) {
-    RaftClient client = createClient();
+    RaftClient client = createClient(TimeDuration.valueOf(35, TimeUnit.SECONDS),
```

Comment:
These are better defined as a property, so that we can tune for when network is congested.

Reply:
Added a `PropertyKey` for each.